docs/streams.html

```diff
@@ -808,6 +808,12 @@
         <h2><a id="streams_upgrade_and_api" href="#streams_upgrade_and_api">Upgrade Guide and API Changes</a></h2>
 
         <p>
+        If you want to upgrade from 0.10.2.x to 0.11.0 you don't need to do any code changes as the API is fully backward compatible.
```

Comment:
Streams did not remove any deprecated APIs AFAIK -- if I missed one, please point it out as we should explicitly list what was removed.

And for internal APIs -- well, they are internal, no reason to talk about them IMHO.